### ISSUE TITLE
`<Dropdown>` is not accessible by keyboard

### ISSUE THREAD

**Issue by tulup-conner** [themesberg/flowbite-react#648]
**Describe the bug**
`<Dropdown>`s should gain focus, so that the next Tab is the first element in the dropdown.

**To Reproduce**

1. Go to https://flowbite-react.com/dropdown
2. Tab to a Dropdown and hit Enter to expand
3. Notice you cannot tab or use keyboard to navigate dropdown items
